config/locales/en.yml

```diff
@@ -1089,7 +1089,7 @@ idv.failure.phone.warning.next_steps_html: Try <strong>another</strong> number t
 idv.failure.phone.warning.try_again_button: Try another number
 idv.failure.phone.warning.you_entered: 'You entered:'
 idv.failure.sessions.exception: There was an internal error processing your request.
-idv.failure.sessions.fail_html: For your security, we limit the number of times you can attempt to verify personal information online. <strong>Try again in %{timeout}.</strong>
+idv.failure.sessions.fail_html: To prevent fraud, we limit the number of times someone can attempt to verify a document online. <strong>Try again in %{timeout}.</strong>
```

Comment:
Could we double check that this text change applies to doc auth only, and not other instances of rate limited screens like the Verify your Info or Verify Phone steps? I see that the previous text says "personal information" and is updated to "document" here

Feel free to screenshot the personal info and phone numbers, or happy to pair and review if that's easier. And lmk if we're missing any translated text strings